src/functional/fencil_processors/callables/cpp/build.py

```diff
@@ -93,8 +93,8 @@ def visit_FindDependency(self, dep: FindDependency):
                 return textwrap.dedent(
                     """\
                     FetchContent_Declare(GridTools
-                        GIT_REPOSITORY https://github.com/GridTools/gridtools.git
-                        GIT_TAG        master
+                        GIT_REPOSITORY https://github.com/havogt/gridtools.git
```

Comment:
I added support for pybuffer as neighbortable in GridTools/gridtools, but not sure if we should keep it there or add it to GT4Py.